Sources/NostrSDK/Filter.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  Filter.swift
+//  
+//
+//  Created by Joel Klabo on 5/26/23.
+//
+
+import Foundation
+
+struct Filter {
```

Comment:
I'm hoping that we might be able to have this raw filter private to the library, and expose a filter type to the client app that uses native types, such as a `NostrKind` enum for the `kinds` part of the filter and `Keypair` types for the `pubkeys`. It could be converted the raw type when making requests, but that can come later.

Reply:
What do you think we should for this right now? I think when we start setting up the public API we should definitely do that but not sure if we need to now. What do you think?